pyproject.toml

```diff
@@ -27,9 +27,8 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "jupyter_ydoc>=1.1.0,<2",
-    "y-py>=0.6.0,<0.7",
```

Comment:
We should probably add pycrdt here

Reply:
I wanted to do this, but pinning a version of `pycrdt` is quite tricky since it is a dependency of a dependency of `jupyter-collaboration`, which is not a direct dependency of `jupytercad-freecad`

Reply:
should we add `pycrdt` without version constraints?

Reply:
ok. Well it's not too bad to not make it a dependency (until it is). Let's keep it that way for now.